core_data_modules/traced_data/io.py

```diff
@@ -323,6 +324,60 @@ def import_coda_to_traced_data_iterable(user, data, key_of_raw, scheme_keys, f,
 
                 td.append_data({key_of_coded: code}, Metadata(user, Metadata.get_call_location(), time.time()))
 
+    @staticmethod
+    def import_coda_to_traced_data_iterable_as_matrix(user, data, key_of_raw, coda_keys, f, key_of_coded_prefix=""):
```

Comment:
Could you setup a list infrastructure priorities? I think I would like to have Coda be able to use TraceData directly fairly soon - but we should review what we've learnt about TraceData before we do that.